### ISSUE TITLE
Mist won't start - Stuck on "Ethereum node connected" squash image

### ISSUE THREAD

**Issue by vitormessias** [ethereum/mist#3113]
I installed the latest version of mist on my HDD computer, which had the problem for synching. As the solution for the isssue 2508 states that I should use the new feature Synch with light client (beta) to solve it - I did just that. Initially it solved because my ether arrived to my account, but after closing the application, I can't get back to it because it is frozen on the squash page saying "Ethereum node connected". It doesn't show me the "Lanch application" option as before.

What should I do? I need to make a transfer.

I appreciate any help.

- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.9.1`
OS & Version: windows 8.1 64bit
Node version: `geth 1.7.0` 
Number of blocks synchronized: fully synch (I can't see it because application won't start)
```

**Comment by bertolo1988** [comment]
I have been having the same issue. I am using v0.9.1

**Comment by sprkoolguy** [comment]
Add me to the list, same problem, tried reinstalling 0.9.1 again to no effect.

**Comment by Akiryx** [comment]
Same as the above. Really need to have access to my wallets and such right now and this is kinda screwing me.

**Comment by bblboy54** [comment]
I just started having this same issue today using 0.9.1....  I launched the application and that splash screen came up along with the window notifying me to update to 0.9.2....  after it didn't work, I tried restarting it... same thing.....  I then installed the update and tried opening again and it's still doing the same thing.  I had been using the light sync for a few weeks with no issue until today.  I'm using this on Ubuntu 16.04.3 (64 bit)

**Comment by bdjukic** [comment]
One workaround for this issue is to launch the client in the normal mode and wait for the wallet UI to appear. Once that happens, you can switch to Light client mode and wait for sync to complete.

**Comment by yomon598** [comment]
Bug still here.As someone else mentioned, this started happening after the notification to upgrade to 0.9.3 and persists after the update.

**Comment by dlezama** [comment]
For me the solution was waiting :) I tried going back to the last version and other different things. Then I just tried to leave it there and after a few minutes it started. Next time it started fast, maybe it was doing some check or repair?

**Comment by ManuelS89** [comment]
Same problem here on updating to 0.9.3.

**Comment by atreyyu** [comment]
The same problem after updating to v0.9.3. Also very few peers (0-2) and the application refuses to sync sometimes for minutes/hours even if there is one peer connected.

**Comment by bertolo1988** [comment]
Same problem on version 0.9.3

Could some dev answer this?

**Comment by yuvalgut** [comment]
Same here.
Running on Win 10 machine. It started right after the notifications about the update, and it still there after the update to 0.9.3
![image](https://user-images.githubusercontent.com/11994190/33450025-3486771e-d613-11e7-936e-616ad7aed519.png)

**Comment by ma68546** [comment]
I have the same mini-window as yuvalgut (one post above) 

But after three hours or so it suddenly showed the "launch application" button and the text switched to "Ethereum node needs to sync, please wait ... looking for peers..."
probably it is just some kind of super slow connection issue thing?

**Comment by apatsekin** [comment]
Same issue. Ethereum Wallet 0.9.3 on MacOS 10.12.6
At first it was 1 to 0 peers. Now it just stuck on this message.

**Comment by dcnl1980** [comment]
Yes same here! First problems with tokens and now it's not even connecting. Come on, make some good working Ethereum wallet for us!

**Comment by dcnl1980** [comment]
I have MacOS High Sierra and latest Ethereum Wallet, also Mist hangs on this one.

**Comment by yuvalgut** [comment]
Just to let you know, after some hours it did open the gui with the wallets, just like other ppl said(@ma68546 , @dlezama )

**Comment by GiorgioG** [comment]
bump

**Comment by bozgoren** [comment]
I was waiting for days to get payment from mining pool. when i get it, mist is stuck with this screen. And no solution here.

**Comment by Munzey** [comment]
same issue. On one occasion it did open after fully syncing, but the amount in one of my accounts was not displayed correctly (etherscan showed the balance i was expecting)

**Comment by tndog** [comment]
Did what @bblboy54 suggested - launch from terminal - it fixes the problem.  But I can't connect to any peers yet...

**Comment by danjpark** [comment]
Launched from Windows Powershell and I got the following msg:
```
PS F:\__Crypto\Ethereum> & '.\Ethereum Wallet.exe'
PS F:\__Crypto\Ethereum>
[2017-12-08 07:51:41.527] [INFO] main - Running in production mode: true
[2017-12-08 07:51:41.575] [INFO] EthereumNode - undefined 'light' 'fast'
[2017-12-08 07:51:41.576] [INFO] EthereumNode - Defaults loaded: geth main light
[2017-12-08 07:51:42.250] [INFO] main - Starting in Wallet mode
[2017-12-08 07:51:42.274] [INFO] Db - Loading db: C:\Users\danpark\AppData\Roaming\Ethereum Wallet\mist.lokidb
[2017-12-08 07:51:42.286] [INFO] Windows - Creating commonly-used windows
[2017-12-08 07:51:42.287] [INFO] Windows - Create secondary window: loading, owner: notset
[2017-12-08 07:51:42.333] [INFO] updateChecker - Check for update...
[2017-12-08 07:51:45.789] [INFO] Windows - Create primary window: main, owner: notset
[2017-12-08 07:51:45.805] [INFO] Windows - Create primary window: splash, owner: notset
[2017-12-08 07:51:46.306] [INFO] ipcCommunicator - Backend language set to:  en
[2017-12-08 07:51:46.859] [INFO] (ui: splashscreen) - Web3 already initialized, re-using provider.
[2017-12-08 07:51:46.937] [INFO] (ui: splashscreen) - Meteor starting up...
[2017-12-08 07:51:46.986] [INFO] ClientBinaryManager - Initializing...
[2017-12-08 07:51:46.987] [INFO] ClientBinaryManager - Checking for new client binaries config from: https://raw.githubusercontent.com/ethereum/mist/master/clientBinaries.json
[2017-12-08 07:51:47.103] [INFO] ClientBinaryManager - Initializing...
[2017-12-08 07:51:47.103] [INFO] ClientBinaryManager - Resolving platform...
[2017-12-08 07:51:47.103] [INFO] ClientBinaryManager - Calculating possible clients...
[2017-12-08 07:51:47.105] [INFO] ClientBinaryManager - 1 possible clients.
[2017-12-08 07:51:47.105] [INFO] ClientBinaryManager - Verifying status of all 1 possible clients...
[2017-12-08 07:51:47.106] [INFO] ClientBinaryManager - Verify Geth status ...
[2017-12-08 07:51:47.255] [INFO] ClientBinaryManager - Checking for Geth sanity check ...
[2017-12-08 07:51:47.255] [INFO] ClientBinaryManager - Checking sanity for Geth ...
[2017-12-08 07:51:47.401] [INFO] Sockets/node-ipc - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2017-12-08 07:51:47.404] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2017-12-08 07:51:48.407] [INFO] updateChecker - App (0.9.1) is out of date. New v0.9.3 found.
[2017-12-08 07:51:48.412] [INFO] Windows - Create popup window: updateAvailable
[2017-12-08 07:51:48.413] [INFO] Windows - Create secondary window: updateAvailable, owner: notset
[2017-12-08 07:51:48.440] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2017-12-08 07:51:49.230] [INFO] (ui: popupWindow) - Web3 not yet initialized, doing so now with HttpProvider.
[2017-12-08 07:51:49.322] [INFO] (ui: popupWindow) - Meteor starting up...
[2017-12-08 07:51:49.440] [WARN] Sockets/node-ipc - Connection failed, retrying after 1000ms...
[2017-12-08 07:51:50.404] [ERROR] Sockets/node-ipc - Connection failed (3000ms elapsed)
[2017-12-08 07:51:50.405] [WARN] EthereumNode - Failed to connect to node. Maybe it's not running so let's start our own...
[2017-12-08 07:51:50.405] [INFO] EthereumNode - Node type: geth
[2017-12-08 07:51:50.405] [INFO] EthereumNode - Network: main
[2017-12-08 07:51:50.405] [INFO] EthereumNode - SyncMode: light
[2017-12-08 07:51:50.406] [INFO] EthereumNode - Start node: geth main light
[2017-12-08 07:51:50.407] [INFO] EthereumNode - Start node using C:\Users\danpark\AppData\Roaming\Ethereum Wallet\binaries\Geth\unpacked\geth.exe
[2017-12-08 07:51:53.564] [INFO] EthereumNode - 3000ms elapsed, assuming node started up successfully
[2017-12-08 07:51:53.564] [INFO] EthereumNode - Started node successfully: geth main light
[2017-12-08 07:51:53.567] [INFO] Sockets/node-ipc - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2017-12-08 07:51:53.568] [INFO] Sockets/node-ipc - Connected!
[2017-12-08 07:51:53.569] [INFO] NodeSync - Ethereum node connected, re-start sync
[2017-12-08 07:51:53.569] [INFO] NodeSync - Starting sync loop
[2017-12-08 07:51:53.570] [INFO] Sockets/3 - Connect to {"path":"\\\\.\\pipe\\geth.ipc"}
[2017-12-08 07:51:53.571] [INFO] Sockets/3 - Connected!
[2017-12-08 07:51:53.571] [INFO] main - Connected via IPC to node.
[2017-12-08 07:51:53.609] [INFO] (ui: splashscreen) - Network is mainnet
[2017-12-08 07:51:53.611] [INFO] (ui: splashscreen) - Network is mainnet
[2017-12-08 07:51:55.573] [ERROR] NodeSync - Node crashed while syncing? TypeError: Cannot read property 'number' of undefined
    at ethereumNode.send.then.ret2 (F:\__Crypto\Ethereum\resources\app.asar\modules\nodeSync.js:143:68)
    at tryCatcher (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
[2017-12-08 07:51:55.577] [ERROR] NodeSync - Sync error TypeError: Cannot read property 'number' of undefined
    at ethereumNode.send.then.ret2 (F:\__Crypto\Ethereum\resources\app.asar\modules\nodeSync.js:143:68)
    at tryCatcher (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
[2017-12-08 07:51:55.581] [ERROR] main - Error syncing node TypeError: Cannot read property 'number' of undefined
    at ethereumNode.send.then.ret2 (F:\__Crypto\Ethereum\resources\app.asar\modules\nodeSync.js:143:68)
    at tryCatcher (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
[2017-12-08 07:51:55.581] [INFO] NodeSync - Sync loop ended
[2017-12-08 07:51:55.582] [ERROR] main - Error starting up node and/or syncing TypeError: Cannot read property 'number' of undefined
    at ethereumNode.send.then.ret2 (F:\__Crypto\Ethereum\resources\app.asar\modules\nodeSync.js:143:68)
    at tryCatcher (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:512:31)
    at Promise._settlePromise (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:569:18)
    at Promise._settlePromise0 (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:614:10)
    at Promise._settlePromises (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\promise.js:693:18)
    at Async._drainQueue (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:133:16)
    at Async._drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:143:10)
    at Immediate.Async.drainQueues (F:\__Crypto\Ethereum\resources\app.asar\node_modules\bluebird\js\release\async.js:17:14)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
[2017-12-08 07:52:42.497] [INFO] Windows - All primary windows closed/invisible, so quitting app...
[2017-12-08 07:52:42.498] [INFO] main - Defer quitting until sockets and node are shut down
[2017-12-08 07:52:42.503] [INFO] Sockets - Destroy all sockets
[2017-12-08 07:52:42.504] [INFO] Sockets/3 - Disconnecting...
[2017-12-08 07:52:42.506] [INFO] Sockets/node-ipc - Disconnecting...
[2017-12-08 07:52:43.009] [INFO] NodeSync - Ethereum node stopping, so stop sync
[2017-12-08 07:52:43.009] [INFO] ipcProviderBackend - Ethereum node stopping, disconnecting sockets
[2017-12-08 07:52:43.011] [INFO] EthereumNode - Stopping existing node: geth main
[2017-12-08 07:52:43.058] [INFO] main - About to quit...
[2017-12-08 07:52:43.064] [INFO] Windows - All primary windows closed/invisible, so quitting app...
[2017-12-08 07:52:43.084] [INFO] Windows - All primary windows closed/invisible, so quitting app...```

**Comment by ja72topu** [comment]
I get the same problem today

**Comment by ja72topu** [comment]
so my solution is not to set it in "sync in light client".
It may help you

**Comment by danjpark** [comment]
Problem is, I can't get into the GUI so I can't unset that setting.

**Comment by ja72topu** [comment]
@danjpark download the installer not the zip version and try more times (4 or 5 times) , everytime wait for 3 minuets. It may be work.

**Comment by danjpark** [comment]
Nice. I'm in.

For those who haven't tried using the installer, try that. It worked for me.

**Comment by yugui923** [comment]
Same here
![image](https://user-images.githubusercontent.com/8936346/33864080-ed3b255e-deb8-11e7-9af9-a5dacfda98f6.png)

**Comment by betoharres** [comment]
Same here, OSX Sierra
I updated the wallet and now it wont start

**Comment by davidsanchezvasquez** [comment]
same problem as @yugui923 using Windows 7.

**Comment by KingLegend** [comment]
+1

**Comment by ravigill** [comment]
+1

**Comment by mcna72** [comment]
Clean up your hard disk space. That will solve the problem.

**Comment by GuillaumeKG** [comment]
you can disabled the light sync option by removing  the content of file "C:\Users\[UserName]\AppData\Roaming\Ethereum Wallet\syncmode.
It should resolve the problem as long as you won't re-enable the light sync mode.

**Comment by dreamwagon** [comment]
It appears that the chain is growing faster than the application can load so it appears to be stuck in an endless loop of trying to catch-up and appears to be getting 'stuck'. Just having the option to sync to point in time would be good enough. I don't see how syncing later would resolve this issue as it would still require a full sync to see your balance updated.

**Comment by giantpune** [comment]
Man.  If its not the endless sync issue, its this one where the UI never shows up.  What good is ethereum if I cant open the wallet to send somebody money?  This is an embarrassment.

**Comment by bertolo1988** [comment]
@fulldecent we can't really know.

Why not remove the splashscreen entirely and just open the app and there provide proper feedback of wha is happening.

**Comment by giantpune** [comment]
@fulldecent i waited the time and let it sync and had a functioning ethereum wallet.  Then like an idiot, I closed it and tried to open it 2 days later, expecting it to work.  It refused to open and I google the issue and ended up here in this thread.  I really hope you aren't expected to wait 7 days every time you want to open your ethereum wallet.

**Comment by fulldecent** [comment]
Thanks for the discussion.

This issue, and several duplicates, are clearly describing the problem. There are several solutions to the problem. So I am advocating that visitors here decide which solution they think is appropriate and then go follow that issue and help develop it with your comments. I recommend #3476, which guarantees zero lag on first run and every run. If you are interested, please make a comment or subscribe on that issue.

**Comment by kevinbright** [comment]
Mist and Ethereum Wallet seriously suck.  Took me three days of non-stop issues and I never got them to work.  I ended up importing my private keys on MyEtherWallet.com.  Kind of risky due to fishing issues, but just triple check the URL and security certificate.  Once I uploaded my private key, I could send to my new Ledger address.

**Comment by bertolo1988** [comment]
@kevinbright do something to improve them.

**Comment by fulldecent** [comment]
@kevinbright It seems like you are not a developer and you wont code and fix the solution. No worry, you can still carefully evaluate potential solutions to this problem and contribution to the discussion. Issue aren't for bitching, they are for locking down solutions and implementing the solutions.

Here's further reading https://github.com/ethereum/mist/issues/3476. You can decide if that proposed solution sounds good to you. If yes, subscribe and/or comment and then help raise awareness of that issue to people that are affected.

**Comment by MaxVilchevsky** [comment]
solved. turn off "Sync with Light client(beta)"[http://jmp.sh/fR9Cmp1](url)

**Comment by liushuigs** [comment]
+1 same error Mist 0.9.3 and geth 1.8.1

**Comment by bunnyext** [comment]
Hello Guy's Uninstall geth from your computer and reinstall it on its default directory C:\Program Files\Geth
after that Wallet will show the Application launch option again. Works for me. 🥇

**Comment by biotin** [comment]
Uninstall geth and reinstall it (Used with brew on my MacOs Sierra 10.13.3.) Then run in terminal "geth --light" and then start Ethereum Wallet. Worked for me. Ethereum Wallet version 0.9.3, geth version 1.8.2.

**Comment by cudapop** [comment]
Just leave the wallet running, it will complete it's synch loop check and then proceed to open the actual application window. Here's a recent example in my case: the log would show the following as final entries when it "hangs" at the splash screen:

[2018-04-13T16:46:37.114] [INFO] NodeSync - Ethereum node connected, re-start sync
[2018-04-13T16:46:37.151] [INFO] NodeSync - Starting sync loop
[2018-04-13T16:46:37.156] [INFO] main - Connected via IPC to node.
[2018-04-13T16:46:37.229] [INFO] Sockets/4 - Connected!
[2018-04-13T16:46:37.231] [INFO] Sockets/2 - Connected!
[2018-04-13T16:46:37.469] [INFO] (ui: splashscreen) - Network is mainnet
[2018-04-13T16:46:37.478] [INFO] (ui: splashscreen) - Network is mainnet

It then sat there for around 20 mins before it updated the logs with:

[2018-04-13T17:06:15.459] [INFO] NodeSync - No more sync necessary
[2018-04-13T17:06:15.463] [INFO] NodeSync - Sync loop ended

After which the main app window appeared and everything was normal.

This was tested on the latest release, Ethereum-Wallet-linux64-0-10-0.zip.

**Comment by Poincare1** [comment]
Bump.

+4 hours on this screen. Nothing is happening. 

![image](https://user-images.githubusercontent.com/8528036/38774306-902cacb4-402b-11e8-9f50-7cfef4dfdbc1.png)

**Comment by mrsanchos** [comment]
The same issue. MacOS Sierra
<img width="401" alt="screen shot 2018-04-15 at 10 05 48" src="https://user-images.githubusercontent.com/23494254/38775826-9b1415fe-4094-11e8-9aae-d0f999781c66.png">

**Comment by hungpham2411** [comment]
Seems Ethereum Wallet got some updates a few day ago and the problem has gone on my (even though if you run on Terminal then node-rpc connection failed is still there).
Now, after Ethereum Wallet download all the blocks, actual window will automatically open.

**Comment by mrsanchos** [comment]
@hungpham2411 I tired fight with Misp. So I exported private key and moved all my funds to another wallet.

**Comment by Emobe** [comment]
@mrsanchos I just got this today after updating my Arch OS. Seems like a recent bug.

**Comment by mrsanchos** [comment]
@Emobe I've started use Exodus and I really like it. Also I don't need more store all transactions in my macbook pro. So I am happy with it.

**Comment by Emobe** [comment]
Just a quick update. It EVENTUALLY opens! After like 10-15 minutes of waiting, the application opens properly. I recall this happening before too.

**Comment by k-vladyslav** [comment]
Same here, Mac OS High Sierra. It used to work w/o any issues and then suddenly stopped to launch, just hanging on "Ethereum node connected" splash screen :(

**Comment by biotin** [comment]
I thinks its take 5-15 min. After this time its connect and work properly. Just wait. (You must have  new versions  geth and wallets).

**Comment by ncsolar** [comment]
Same issue here waited a couple hours OS X 10.12.6

**Comment by mushketyk** [comment]
Finally, it worked for me. I've disabled the light mode (Develop -> Sync with Light client) and then waited for a couple of minutes.

**Comment by tessr8** [comment]
I love how there is NOT one real solution! Try this or try that, uninstall then...  Look folks it's the Wild West! I have never had any consistent launch EVER! Sometimes it works and...  For me I must launch it from the command line "Ethereum Wallet" --syncmode=fast. Then when it launches I switch back to light client from the GUI.

**Comment by scionGbh** [comment]
Hi there, ok, I had this issue more than once. in my case the problem came mostly when the update geth window appear and I click to update.

It seams that the current network get out of sync.

To solve it what I do is change my network from main to rinkeby or ropsten and then the GUI come up, then once I am in the GUI I change back to main network.

To change the network you can go to the Ethereum Wallet folder in AppData (Windows) or copy and paste this path on any window url bar. **%appdata%/Ethereum Wallet**
and then find the file **network** you can open that with any text editor, in my case **Notepad++**
then change the text **main** with other network like **rinkeby**
Open the wallet as you do it normally, click on the icon, if you have luck you will get the GUI, and then go back to main network by selecting **Develop -> Network -> Main Network** or you can use the GUI shortcut **Ctrl+Alt+1**.

I hope it works with some one else, for me always works :)

**Comment by evertonfraga** [comment]
Hi all, we've been working hard on solutions to sync problems — we know it's painful to get ethereum to sync on *full mode.*

On the recently released version 0.11.0, you can get connected almost instantly, while syncing in the background with the Light Client.

Version 0.11.0:
https://github.com/ethereum/mist/releases/tag/v0.11.0
